proto/cosmos/base/crypto/v1beta1/crypto.proto

```diff
@@ -2,33 +2,9 @@ syntax = "proto3";
 package cosmos.base.crypto.v1beta1;
```

Comment:
@blushi would it actually make sense to remove this proto file, and put the 2 remaining messages inside `cosmos.crypto.multisig`?

Reply:
yes I believe that makes sense.